cli.js

```diff
@@ -8,8 +8,9 @@ const cli = meow(`
 	  $ wait-for-localhost [port]
 
 	Options
-	  --use-get  Use the HTTP-method GET instead of HEAD to test if the server is ready
-	  --path     Use a custom path. For example, /health for a health-check endpoint.
+	  --use-get       Use the HTTP-method GET instead of HEAD to test if the server is ready
+	  --path          Use a custom path. For example, /health for a health-check endpoint.
+	  --status-codes  Define status codes indicating the server is ready. 
```

Comment:
I think it should accept `--status-codes=200,201`, because otherwise, it is not clear that `--status-codes=201` would override `200`. Meow doesn't support comma-separated, so you can just accept a string in Meow and split it on comma yourself.